include/swift/Runtime/SwiftDtoa.h

```diff
@@ -37,6 +37,10 @@
  #endif
 #endif
 
+#if defined(__ANDROID__) && (defined(__i386) || defined(__x86_64__))
+ #undef SWIFT_DTOA_FLOAT80_SUPPORT
+#endif
+
```

Comment:
Nit: I would create a block above, since `SWIFT_DTOA_FLOAT80_SUPPORT` is determined there for other platforms. Android is probably falling through `|| defined(__linux__)`, so maybe adding `#elif defined(__ANDROID__)` before that one is enough. I will also remove the checks for the architecture in that case, since I don't think Android has Float80 in any architecture.

Reply:
Let me check that "if/else" checks to avoid `#undef`...

Reply:
True. Float80 needs to be disabled for any Android architecture.